i6_models/parts/conformer/mhsa_rel_pos.py

```diff
@@ -0,0 +1,248 @@
+from __future__ import annotations
+
+
+__all__ = ["ConformerMHSARelPosV1", "ConformerMHSARelPosV1Config"]
+
+from dataclasses import dataclass
+import math
+from typing import Optional
+
+import torch
+from torch import nn
+import torch.nn.functional as F
+
+from i6_models.config import ModelConfiguration
+from i6_models.util import compat
+
+
+@dataclass
+class ConformerMHSARelPosV1Config(ModelConfiguration):
+    """
+    Attributes:
+        input_dim: input dim and total dimension for query/key and value projections, should be divisible by `num_att_heads`
+        num_att_heads: number of attention heads
+        att_weights_dropout: attention weights dropout
+        learnable_pos_emb: whether to use learnable relative positional embeddings instead of fixed sinusoidal ones
+        rel_pos_clip: maximal relative postion for embedding
+        with_pos_bias: whether to add additional position bias terms to the attention scores
+        pos_emb_dropout: dropout for the positional embeddings
+        dropout: multi-headed self attention output dropout
+        dropout_broadcast_axes: string of axes to which dropout is broadcast, e.g. "T" for broadcasting to the time axis
+                                setting to None to disable broadcasting
+    """
+
+    input_dim: int
+    num_att_heads: int
+    att_weights_dropout: float
+    dropout: float
+    learnable_pos_emb: bool = True
```

Comment:
This default is different from Trafo XL (sorry, my bad that I wrote it wrong earlier). But also: I thought that we said we do not want any defaults? So why here? But then, what default do you want? To what is commonly used (ESPnet for Conformer, or also Trafo-XL, etc?), then it should be False. Or what we had before?
```suggestion
    learnable_pos_emb: bool = False
```